code/modules/mob/living/carbon/human/species_types/plasmamen.dm

```diff
@@ -15,7 +15,9 @@
 	blacklisted = 1 //See above
 	burnmod = 1.5
 	heatmod = 1.5
+	brutemod = 1.5
 	breathid = "tox"
+	speedmod = 1

```

Comment:
And add this?

Reply:
both are accidents because my branch is old and I was under the influence